elab_bridge/tests/test_server_interface.py

```diff
@@ -26,11 +26,18 @@ def test_upload_experiment(initialize_test_dir):
 
 def test_download_experiment(initialize_test_dir):
     json_file = test_directory / 'testfiles_elab' / 'downloaded_experiment.json'
+    upload_experiment_file = test_directory / 'testfiles_elab' / 'experiment.json'
+    upload, experiment_id = upload_experiment(server_config_json=SERVER_CONFIG_YAML,
+                               experiment_file=upload_experiment_file,
+                               experiment_title='UploadExperiment')
+
     experiment = download_experiment(save_to=json_file,
                                      server_config_json=SERVER_CONFIG_YAML,
-                                     experiment_id=232,
+                                     experiment_id=experiment_id,
                                      format='json')
 
+    assert upload == experiment
+
```

Comment:
Ideally we would also remove the experiment again from the server here. Does the API support deleting of experiments and templates? Then this would be the place to use it.